### DISCUSSION TITLE
Is there any room for adding docs reference to sea_query::LockType?

### DISCUSSION THREAD

**Discussion by k0i** [SeaQL/sea-orm#980] [Ideas]
Hello!

I'm trying to build a toy CRUD application using sea-orm.
Although I feel sea-orm is very handy and user-friendly, 
I hit a bump in the road when I tried to write `'SELECT... FOR UPDATE'` query. 

I couldn't find a reference to the above query in sea-orm docs...
But I managed to find `sea_query::LockType` in crate.io and finally achieved it.

I would like to ask to the community if there is any chance to add references about LockType on the docs.

```rust
use sea_orm::{
    sea_query::LockType, ...
}
...
let cake: Option<cake::Model> =Cake::find_by_id(1)
            .lock(LockType::Update)
            .one(db)
            .await?
```

**Comment by billy1624** [comment]
Hey @k0i, thanks for asking!! I'd say a PR would be welcomed! It's definitely worth documenting and we should :)

**Reply by k0i** [reply]
Thank you for your reaction,billy1664!!!
I'll create one in a few days!!

Thank you for your time! 🙇 🙇
